set.mm

```diff
@@ -99209,18 +99209,6 @@ our Axiom of Choice (in the form of ~ ac2 ).  The proof does not depend
     wa domtr ) ACEBDERZAABFZGHZUAABIZGHAUCGHTUAJEAUAKUBABCDLABMAUAJNOABCDPAUAUC
     SQ $.
 
-  $( Cartesian product dominates disjoint union for sets with cardinality
-     greater than 1.  Similar to Proposition 10.36 of [TakeutiZaring] p. 93.
-     (Contributed by Mario Carneiro, 18-May-2015.)  Use ~ djuxpdom instead.
-     (New usage is discouraged.) $)
-  cdaxpdom $p |- ( ( 1o ~< A /\ 1o ~< B ) -> ( A +c B ) ~<_ ( A X. B ) ) $=
```

Comment:
The description of this commit should be "Remove..."

Reply:
Thanks; fixed.